spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -16,3 +16,74 @@ We're also more than happy to extend this section; If you want to add a '`how-to
 can send us a {github-code}[pull request].
 --
 
+== Configure Remote Maven Repository
+
+If you'd like to connect to your organization's internal maven repository, you can override the default 
+repository location by passing the credentials via the following environment properties:
+
+```
+export MAVEN_LOCAL_REPOSITORY=mylocalMavenRepo
+export MAVEN_REMOTE_REPOSITORIES_REPO1_URL=https://repo1
+export MAVEN_REMOTE_REPOSITORIES_REPO1_AUTH_USERNAME=user1
+export MAVEN_REMOTE_REPOSITORIES_REPO1_AUTH_PASSWORD=pass1
+export MAVEN_REMOTE_REPOSITORIES_REPO2_URL=https://repo2
+```
+
+If you are running behind a proxy, then you need to specify the proxy settings via environment properties 
```

Comment:
We deliberately removed the following from the previous doc and have only `SPRING_APPLICATION_JSON` as the way to set the environment properties.